libs/barista-components/tag/src/tag-list/tag-list.spec.ts

```diff
@@ -106,6 +106,26 @@ describe('DtTagList', () => {
       expect(moreBtn.nativeElement.textContent).toBe('1 More...');
     });
 
+    it('should not show a less button when all tags fit into the viewport', () => {
+      const lessBtn = fixture.debugElement.query(
+        By.css('.dt-tag-list-less-btn'),
+      );
+      expect(lessBtn).toBeNull();
+    });
+
+    it('should show less button when a tag does not fit into viewport and list is expanded', () => {
```

Comment:
Could you add a test to check that the counter for showMore is correctly updated when adding a tag while tag list is expanded?